pybamm/parameters/parameter_sets.py

```diff
@@ -49,6 +56,15 @@
     "experiment": "1C_discharge_from_full_Kim2011",
 }
 
+Chen2020 = {
+    "chemistry": "lithium-ion",
+    "cell": "LGM50_Chen2020",
+    "anode": "graphite_Chen2020",
+    "separator": "separator_Chen2020",
+    "cathode": "nmc_Chen2020",
+    "electrolyte": "lipf6_Nyman2008",
+    "experiment": "1C_discharge_from_full_Chen2020",
+}
```

Comment:
take a look at the latest master (see PR #818)-- you can add a reference here so people know to cite your paper if they use your parameter set

Reply:
I talked to @tinosulzer and he said he would do so after the PR is merged.